connector/avro/src/test/scala/org/apache/spark/sql/execution/benchmark/AvroReadBenchmark.scala

```diff
@@ -87,7 +87,7 @@ object AvroReadBenchmark extends SqlBasedBenchmark {
 
         prepareTable(
           dir,
-          spark.sql("SELECT CAST(value AS INT) AS c1, CAST(value as STRING) AS c2 FROM t1"))
+          spark.sql(s"SELECT value % ${Int.MaxValue} AS c1, CAST(value as STRING) AS c2 FROM t1"))
```

Comment:
This looks like a different benchmark, why do we use a static value here, @yaooqinn ?

Reply:
Oh, nvm.